secrets/localsecrets/localsecrets_test.go

```diff
@@ -63,6 +63,35 @@ func (v verifyAs) ErrorCheck(k *secrets.Keeper, err error) error {
 	return nil
 }
 
+func TestSmallData(t *testing.T) {
+	tests := [][]byte{
+		nil,
+		{},
+		{0},
```

Comment:
What's the difference between `{0}` and `{65}` as far as this function is concerned? You may want to test a larger value like nonceSize-1 instead?

Reply:
Good idea; done.